Automattic/wp-calypso
# Stats: Adding Card info component

Related to https://github.com/Automattic/jetpack-roadmap/issues/1384

## Proposed Changes

* add an icon with a popover to the card header area
* refactor how we use the "new" badge to consolidate similar functionality
* fix property declaration for missing icons

<img width="449" alt="SCR-20240717-lvhr" src="https://github.com/user-attachments/assets/ed0f45ac-d478-4bb5-a2e5-2b5a21567040">


## Why are these changes being made?


* to allow users to access documentation then cards have data

## Testing Instructions



* navigate to the live branch
* verify that cards on the traffic page have an icon and a popover when modules have data
* verify that the icon is missing when cards don't receive data to display
* verify that the "new" badge display didn't change

## Pre-merge Checklist
